Bundle/WidgetBundle/Builder/WidgetFormBuilder.php

```diff
@@ -136,7 +136,7 @@ public function renderNewQuantumForms($slot, View $view, $widgets, $activeWidget
     {
         $forms = [];
         foreach ($widgets as $key => $widget) {
-            $_quantum = null !== $quantum ? $quantum : $key;
+            $_quantum = null !== $quantum ? $quantum : $this->convertToString($key);
```

Comment:
I think Widgets array shouldn't have this `static` key because this key as always been interpreted as the quantum number. I removed it.